samples/bluetooth/central_and_peripheral_hr/README.rst

```diff
@@ -143,11 +146,14 @@ Testing with nRF Connect for Desktop
 
             Heart Rate Measurement Value: 128 bpm
 
-#. The `nRF Connect for Desktop`_ also detects the Central and Peripheral HRS sample Heart Rate Service.
-#. Enable the notification for the **Heart Rate Measurement** characteristic.
-#. Write again value ``06 80`` and click the :guilabel:`Play` button to send a notification.
-   The same value appears for the **Heart Rate Measurement** characteristic.
-   The sample works as relay for the Heart Rate Service.
+      The Bluetooth Low Energy app also detects the Central and Peripheral HRS sample Heart Rate Service.
+
+   #. Enable the notification for the Heart Rate Measurement characteristic.
+   #. Write again value ``06 80`` and click the :guilabel:`Play` button to send a notification.
+
+      The same value appears for the Heart Rate Measurement characteristic.
+
+The sample works now as relay for the Heart Rate Service.
```

Comment:
I don't have strong opinion here but maybe it would look better in generated doc as a main point if instruction. I mean:

#. The sample works now as relay for the Heart Rate Service.

Reply:
No, Kamil, this is not a step, it is the end result of the procedure you did with the steps before.